MapboxDirections/MBRoute.swift

```diff
@@ -182,6 +182,13 @@ open class Route: NSObject, NSSecureCoding {
         }
         return nil
     }
+    
+    /**
+     A unique identifier for a directions request.
+     
+     Each route produced by a single call to `calculate(_:completionHandler:)` has the same route identifier.
+     */
+    open var routeIdentifier: String?
```

Comment:
If these UUIDs are persistent across application sessions, we should encode and decode `routeIdentifier` like the other properties.